MonoGame.Framework/Input/Keyboard.Windows.cs

```diff
@@ -4,21 +4,57 @@
 
 using System;
 using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
 
 namespace Microsoft.Xna.Framework.Input
 {
     public static partial class Keyboard
     {
-        static List<Keys> _keys;
+        private static readonly Keys[] DefinedKeys;
+
+        private static readonly byte[] _keyState = new byte[256];
+        private static readonly List<Keys> _keys = new List<Keys>(10);
+
+        private static bool _isActive;
+
+        [DllImport("user32.dll")]
+        [return: MarshalAs(UnmanagedType.Bool)]
```

Comment:
Yeah, it's not actually required for the 4-byte `BOOL` type (source: https://blogs.msdn.microsoft.com/jaredpar/2008/10/14/pinvoke-and-bool-or-should-i-say-bool/). I'll remove it.